data/transport missions.txt

```diff
@@ -1868,6 +1872,309 @@ mission "Terraforming 5"
+				accept
+	
+	npc assist
+		government "Uninhabited"
+		personality derelict fleeing uninterested waiting pacifist mute
+		ship "Asteroid" "Target Asteroid"
+	
+	on complete
+		payment 200000
+		event "terraforming timer 2" 365 730
+		event "terraforming Tundra"
+		conversation
+			`A plume of smoke forms in a mushroom cloud shape above where the asteroid hit. The scientists are viewing their instruments in anticipation for what happens next.`
+			`	Almost an hour after the impact, the ground begins to shake slightly, even though the volcano is located hundreds of kilometers away. "This might be it!" Nolan exclaims as he watches closely at a seismometer receiving readings from near the impact site. Suddenly the seismometer spikes, and another plume of smoke, mucher larger than the asteroid impact, rises from the impact site. Half an hour later, the low sound of the massive eruption can be heard in the spaceport, triggering another burst of cheers from the scientists. "It actually worked!" one of them yells out.`
+			`	You fly the scientists over top of the volcano, allowing them to take readings of the air and surface. "Atmospheric carbon dioxide levels are at 4000 parts per million. Methane levels at 20 parts per million. As the smoke disperses across the planet, those levels should drop to a tenth of what they are now, warming the entire planet."`
+			`	The heat has also formed a huge circle of melted snow around the volcano, revealing the green color of trees and the dull grays and browns of rock.`
+			choice
+				`	"What do we do now?"`
+					goto what
+				`	"It looks beautiful from up here."`
+			
+			`	"Yes it does, Captain," the Republic representative says. "Yes it does."`
+				goto end
+			
+			label what
+			`	"Now we wait," Amy says to you. "We should be able to tell how this will affect Tundra's climate in a year or two."
+			
+			label end
+			`	You steer your ship back to the spaceport. Amy pays you <payment> after you land. "Thanks for your help, Captain. We're going to stay here and monitor Tundra's climate as it shifts, so there's nothing more that we need you for."`
+
+
+
+event "terraforming Tundra"
+	planet "Tundra"
+		description `Millions of years ago, this was a warm world, perhaps even tropical. But some cataclysmic event, perhaps a meteor strike or a massive volcanic eruption, altered the planet's atmosphere enough to turn it into the nearly lifeless, frozen planet it is today. Recent experiments in terraforming the planet have attempted to remedy the climate by causing a supervolcano to prematurely erupt.`
+		description `	The first settlers came here to drill for the oil trapped deep under the surface, a relic of Tundra's former, more lively days. Instead of refining the oil into plastic, which sells relatively cheap in this region, they have developed an industry in synthetic fabrics and clothing.`
+
+
+
+event "terraforming timer 2"
+
+
+
+mission "Terraforming Follow-up"
+	source
```

Comment:
`landing`?
`minor`?
If not, then you can remove `attributes spaceport` as it can't be triggered when there is no spaceport to visit 😉